ios/LayoutReanimation/REASharedTransitionManager.m

```diff
@@ -629,4 +628,16 @@ - (void)enableCleaningForViewTag:(NSNumber *)viewTag
   }
 }
 
+- (NSDictionary *)prepareDataForWorklet:(NSMutableDictionary *)currentValues
+                           targetValues:(NSMutableDictionary *)targetValues
+{
+  NSMutableDictionary *workletValues = [_animationManager prepareDataForLayoutAnimatingWorklet:currentValues
```

Comment:
I wonder what would happen if we'd fill transform and border radius data for all layout animations. Would it suffice for regular layout animations to support these properties for layout update animations?

Reply:
Yes, it should be enough